calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml

```diff
@@ -0,0 +1,115 @@
+# This policy opens port 53 and 5353 for DNS,
+# port 2049 for communication with NFS file servers,
+# ports 443 and 3260 for communication to block storage,
+# port 10250 for the openVPN pod, port 4443 or 6443 for metrics-server, and
+# port 2040 and 2041 on 172.21.0.0/24 for the master API server local proxy. It also
+# allows pod access to other pods, denies access to private networks but allows access to
+# public networks, and allows access in each zone of the region to the master.
+
+apiVersion: projectcalico.org/v3
+kind: GlobalNetworkPolicy
+metadata:
+  name: allow-egress-pods-private
+spec:
+  egress:
+  - action: Allow
+    destination:
+      ports:
+      - 53
+      - 5353
+      - 443
+      - 2049
+      - 3260
+      - 10250
+    protocol: UDP
+    source: {}
+  - action: Allow
+    destination:
+      ports:
+      - 53
+      - 5353
+      - 443
+      - 4443 # Kubernetes 1.19 or later
+      - 6443 # OpenShift 4.3 or later
+      - 2049
+      - 3260
+      - 10250
+    protocol: TCP
+    source: {}
+  - action: Allow
+    destination:
+      ports:
+      - 2040:2041
+      nets:
+      - 172.20.0.0/24
+    protocol: UDP
+  - action: Allow
+    destination:
+      ports:
+      - 2040:2041
+      nets:
+      - 172.20.0.0/24
+    protocol: TCP
+  - action: Allow
+    destination:
+      selector: projectcalico.org/orchestrator == 'k8s'
+    source: {}
+  # Denies pod egress to private networks. If workers are connected to a public VLAN,
```

Comment:
We should probably change this comment.  We are allowing pod egress to private networks here, not denying

Reply:
Agreed, let's fix this in a separate PR